app/assets/stylesheets/alchemy/icons.scss

```diff
@@ -205,7 +205,7 @@
     background-position: -224px -105px;
   }
 
-  &.page, &.clipboard_pages {
+  &.page, &.node, &.clipboard_pages {
```

Comment:
Selector `clipboard_pages` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own single line